bitnami/grafana-loki/templates/_helpers.tpl

```diff
@@ -154,6 +154,17 @@ Get the promtail configuration configmap.
 {{- end -}}
 {{- end -}}
 
+{{/*
+Get the NGINX configuration configmap.
+*/}}
+{{- define "grafana-loki.gateway.configmapName" -}}
```

Comment:
You're not using this helper anywhere. I guess you should use it at `bitnami/grafana-loki/templates/gateway/deployment.yaml` updating the section below:
```diff
      volumes:
        - name: config
          configMap:
-           name: {{ include "grafana-loki.gateway.fullname" . }}
+           name: {{ include "grafana-loki.gateway.configmapName" . }}
```

Reply:
This is correct.  I knew I forgot something.